lib/manager/npm/post-update/yarn.spec.ts

```diff
@@ -29,7 +29,7 @@ describe(getName(__filename), () => {
     jest.resetAllMocks();
     env.getChildProcessEnv.mockReturnValue(envMock.basic);
   });
-  it.each([['1.22.0']])(
+  it.each([['1.22.0'], ['2.1.0']])(
```

Comment:
how to properly reset fs.readFile ? it's currently line 28 but doesn't cover this each